bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/TabsImpl.java

```diff
@@ -38,16 +38,16 @@
     @ValueMapValue(optional = true)
     private String activeItem;
 
-    private String activeItemPath;
+    private String activeName;
```

Comment:
I'd call it `activeItem`.

Reply:
we already have `activeItem` read from the valueMap. The separate property is the result of checking whether such an item exists as a child resource and if not, returning `null`. We could also just return the `activeItem` as it is with no such check.

Maybe `activeItemName` is better?

Reply:
+1